scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# set up folder access for Rawls and Cromwell SAs, as well as billing and project-owners
+# In order to successfully run this script, you must first `gcloud auth login` as a gcloud user with resourcemanager.projects.setIamPolicy permissions
+
+# todo: convert this to Terraform as part of https://broadworkbench.atlassian.net/browse/CA-1195
+
+set -euxo pipefail
+
+
+ENV=$1
+VALID_ENVS="Valid ENVs:
+        dev
+        qa
+        perf
+        alpha
+        staging
+        prod
+       "
+
+if [[ -z "${ENV}" ]]; then
+  echo "Usage: ./add-folder-roles-to-service-accounts.sh <ENV>"
+  echo "${VALID_ENVS}"
+  exit 1
+else
+  # set FOLDER_ID and SA email based on the env
+  if [[ "${ENV}" == "dev" ]] ; then
+    FOLDER_ID=599966635789 # folder: test.firecloud.org/dev
+  elif [[ "${ENV}" == "qa" ]]; then
+    FOLDER_ID=950154083315 # folder: test.firecloud.org/tools (no QA folder)
+  elif [[ "${ENV}" == "perf" ]]; then
+    FOLDER_ID=1076814209841 # folder: test.firecloud.org/perf
```

Comment:
this one folder is not inteneded to used by CWB, it is the folder which buffer's own perf test is using. I don't think you need this. For Rawsl/Terra's perf test, you may use qualify.fc/org. 
And also, that folder is being deprecated

Reply:
thanks for the heads up about `perf` being used for buffer perf testing! I didn't realize that was its purpose.

I think that only FiaBs use quality.fc.org, and real test envs use test.fc.org, so we probably want the perf env to use a folder in test.fc.org. 

Since the perf folder won't be used for the buffer perf tests, would it be ok for the perf env to use this folder?

Reply:
I think you can take that folder. But you may need to create a sub-folder called `CommunityWorkbench`.

Reply:
cool, looks like the perf/CommunityWorkbench already exists! https://console.cloud.google.com/iam-admin/settings?folder=246142551922

Reply:
Is that will be used for perf Rawls talk to perf RBS? I can add a perf RBS pool using that folder.
Also, I think DevOps is working on setting up quality.firecloud.org folders creation
https://broadworkbench.atlassian.net/browse/DDO-1089

Reply:
I think that would be great. The quality.firecloud.org folder is only used by broad-dsde-qa FIABs and not the perf environment so we should keep those separate. Once DDO-1089 is done I can add the prod and QA folders. There's a ticket to track this

```
# todo: to be created as part of https://broadworkbench.atlassian.net/browse/CA-1194 after completion of https://broadworkbench.atlassian.net/browse/DDO-1089
```